pegasus/sites.v3/code.org/public/teach.haml

```diff
@@ -85,7 +85,7 @@ theme: responsive_full_width
     %hr.dark
     %div.centered
       %h4=hoc_s(:teach_page_free_curriculum_heading_02)
-      %div.flex-container.justify-center.wrap{style: "gap: 1rem"}
+      .button-wrapper{style: "justify-content: center"}
```

Comment:
Could this use the change you just made [here](https://github.com/code-dot-org/code-dot-org/pull/55637)?

Reply:
Yep! I'm expecting to get a merge conflict, and will make sure it has that new version 😄